core/src/scene/sceneLoader.cpp

```diff
@@ -528,6 +525,51 @@ Node SceneLoader::mixStyle(const Mixes& mixes) {
     return styleNode;
 }
 
+Mixes SceneLoader::recursiveMixins(Mixes mixIn, const std::string& styleName, Node styles, UniqueStyles& uniqueStyles) {
+    Mixes mixes;
+    mixes.swap(mixIn);
```

Comment:
Seems like just using the input `mixIn` would be equivalent, am I missing something?